@now/node  error: Cannot find module 'pug'

When using the stable or canary builder, express deployments that depend on pug will run into the following error: `Error: Cannot find module 'pug'`

To reproduce this issue, I made a repo that you can clone and deploy: https://github.com/coetry/wtf-pug

Note that i've included a `test-server.js` file, which works fine locally. Also note that this is not an `ncc` error because if you run `ncc build index.js && cp test-server.js dist && cd dist && node test-server.js`, it will work as expected. This error is only encountered when deploying.

@guybedford I was able to reproduce this issue.

## Steps to reproduce

```sh
git clone https://github.com/coetry/wtf-pug
cd wtf-pug
npm install
ncc build test-server.js
rm -rf node_modules # this step is important
node dist/index.js
```

## Error

When you visit http://localhost:3000 you'll see the following error

```
Error: Cannot find module 'pug'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at new View (/Users/styfle/Desktop/foobar/wtf-pug/dist/index.js:16535:14)
    at Function.render (/Users/styfle/Desktop/foobar/wtf-pug/dist/index.js:3765:12)
    at ServerResponse.render (/Users/styfle/Desktop/foobar/wtf-pug/dist/index.js:13687:7)
    at module.exports.318.app.get (/Users/styfle/Desktop/foobar/wtf-pug/dist/index.js:6966:7)
    at Layer.handle [as handle_request] (/Users/styfle/Desktop/foobar/wtf-pug/dist/index.js:3942:5)
    at next (/Users/styfle/Desktop/foobar/wtf-pug/dist/index.js:14058:13)
```

## Workaround

There is a workaround to get this to work as expected thanks to @TooTallNate 🙌 

Our users would never think about this solution so it's a hack to get around the dynamic require.

```diff
--- a/index.js
+++ b/index.js
@@ -3,6 +3,7 @@ const path = require("path");

 const app = express();

+app.engine("pug", require("pug").__express);
 app.set("view engine", "pug");
 app.set("views", path.join(__dirname, "views"));
```

@guybedford Is this fixable?

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "@google-cloud/firestore": "^0.19.0",
     "@sentry/node": "^4.3.0",
     "@tensorflow/tfjs-node": "^0.3.0",
-    "@zeit/webpack-asset-relocator-loader": "0.3.1",
+    "@zeit/webpack-asset-relocator-loader": "0.3.2",
     "analytics-node": "^3.3.0",
     "apollo-server-express": "^2.2.2",
     "arg": "^2.0.0",

